src/layout/section/itemsSection/itemsSection.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import cc from 'classcat'
+
+export interface ItemsSectionProps {
+  readonly children: JSX.Element | string | React.ReactNode
+  readonly className?: Classcat.Class
+  readonly tag?: JSX.Element
+}
+
+/**
+ * Items Section: display a list of items in a display: flex.
+ * Use with two items.
+ */
+const ItemsSection = (props: ItemsSectionProps) => {
+  const { className, children, tag = <div /> } = props
+  const Tag = tag.type
+
+  return (
+    <Tag role="presentation" className={cc([className])}>
```

Comment:
If it's a section it should manage its left/right spacing (e.g. be centered as one column)
The simplest is probably to wrap it into a kirk BaseSection component

Reply:
Check HighlightSection for a dummy but good example.